commands/blacklist.js

```diff
@@ -30,7 +30,11 @@ exports.run = async (client, message, args) => {
     
     //leave his server
     try {
-      client.guilds.filter(c=>c.owner.id === user.id).leave()
+      if(client.guilds.filter(c=>c.owner.id === user.id) > 0) {
```

Comment:
This is pointless. If there are no servers it won't leave any.

Also, you're comparing an array with an integer.

Reply:
i forgot .size... bot ok removed